core/src/test/java/com/ritense/valtimo/BaseIntegrationTest.java

```diff
@@ -16,51 +16,24 @@
 
 package com.ritense.valtimo;
 
-import com.ritense.valtimo.contract.authentication.UserManagementService;
-import com.ritense.valtimo.contract.mail.MailSender;
 import com.ritense.valtimo.domain.contexts.Context;
 import com.ritense.valtimo.domain.contexts.ContextProcess;
 import com.ritense.valtimo.service.ContextService;
-import org.camunda.bpm.engine.RuntimeService;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
+
 import javax.inject.Inject;
 
 @SpringBootTest
 @ExtendWith(SpringExtension.class)
 @Tag("integration")
 public abstract class BaseIntegrationTest {
 
-    @Inject
-    public RuntimeService runtimeService;
-
     @Inject
     public ContextService contextService;
 
-    @MockBean
-    public UserManagementService userManagementService;
-
-    @MockBean
-    public MailSender mailSender;
```

Comment:
Same as previous comment. Are we sure the test do/can not use a real MailSender if you remove this mock?